roles/container_runtime/defaults/main.yml

```diff
@@ -23,7 +23,7 @@ openshift_docker_ent_reg: 'registry.access.redhat.com'
 
 openshift_docker_options: False  # str
 openshift_docker_log_driver: False  # str
-openshift_docker_log_options: []
+openshift_docker_log_options: "max-size=50m"
```

Comment:
This looks like it should be a list of strings, not just a string.

Reply:
Both should work based on https://github.com/openshift/openshift-ansible/blob/master/roles/container_runtime/defaults/main.yml#L35. It checks for string and list.

Reply:
Yeah, string form is legacy.  Lists are definitely preferred here.

Reply:
Okay got it, changed to list.